src/exception/usage/ArcanistBuildableNotPassingException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * Thrown when the Harbormaster Buildable is not passing.
+ */
+final class ArcanistBuildableNotPassingException extends ArcanistUsageException {
+
+  public function __construct() {
+    parent::__construct(pht('The Harbormaster Buildable is not passing. Use BREAKGLASS in case of emergency.'));
```

Comment:
```suggestion
    parent::__construct(pht('The Harbormaster Buildable is not passing. Land is not permitted unless all CI jobs pass. If you are running custom CI jobs that are intended to be non-blocking, you can mark the buildable non-blocking. See t.uber.com/code-land-policy for more information. For emergencies, use BREAKGLASS by adding it to your revision summary. BREAKGLASS will trigger an audit and is reported monthly to EngLT.'));
```
I think it would be helpful to add more details and/or link to a policy page with more details.